demos/editdialog/editdialog.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	ui "github.com/Viv1k/clui"
+
+	termbox "github.com/nsf/termbox-go"
+)
+
+func CreateBox() {
+	dlg := ui.CreateConfirmationEditDialog(
+		"<c:blue>"+"SearchBox:",
+		"Enter search text. Pressing enter would print your input in debug.log")
```

Comment:
Checking the log is not convenient way for a someone to just watch a demo. I'll explain my thoughts in details in pull request discussion.